src/main/java/com/redhat/devtools/intellij/quarkus/lsp4ij/LSPIJUtils.java

```diff
@@ -248,11 +241,7 @@ public static VirtualFile findResourceFor(URI uri) {
     }
 
     public static VirtualFile findResourceFor(String uri) {
-        try {
-            return VfsUtil.findFileByURL(new URL(uri));
-        } catch (MalformedURLException e) {
-            return null;
-        }
+        return VirtualFileManager.getInstance().findFileByUrl(VfsUtilCore.fixURLforIDEA(uri));
```

Comment:
The fix that you did [PsiUtilsLSImpl.java](https://github.com/redhat-developer/intellij-quarkus/commit/16c76e76e34a4a9535e0ef38289b88a42a5d4f16#diff-78d1994d7563bb33170f619c2358419c78997895ce3284cc2aaa0bfe8827ba79) should use `LSPIJUtils.findResourceFor` to have this fix in one location.

Reply:
done